### ISSUE TITLE
github paes

### ISSUE THREAD

**Issue by jungle-boogie** [dinedal/textql#47]
Hello,

I see you have a git branch for github's pages but I don't know the URL to your github page.

Could you please share it and I would also recommend linking to it from your github page.

Thanks!

**Comment by erasche** [comment]
http://dinedal.github.io/textql/

**Comment by dinedal** [comment]
Yes, more time needs to be spend on documentation here and there as well.
